server/text_generation_server/models/vlm_causal_lm.py

```diff
@@ -173,7 +173,10 @@ def batch_tokenized_inputs(cls, requests, tokenizer, processor, config):
             image_id = 0
             for chunk in chunks:
                 if chunk["type"] == "text":
-                    full_text += "<bos>" + chunk["content"] + "\n"
+                    if config.model_type == "paligemma":
+                        full_text += "<bos>" + chunk["content"] + "\n"
+                    else:
+                        full_text += chunk["content"]
```

Comment:
Can you revert this ? This is already taken care of by the PaliGemmaBatch.

Also we should probably raise an error when the query is not `{image}, {text}`. (single text, single image, image before text)